homeassistant/components/bluesound/__init__.py

```diff
@@ -1 +1,28 @@
 """The bluesound component."""
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers.typing import ConfigType
+
+from .const import DOMAIN
+from .media_player import setup_services
+
+CONFIG_SCHEMA = cv.config_entry_only_config_schema(DOMAIN)
+
+
+async def async_setup_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> bool:
+    """Set up the Bluesound entry."""
+    await hass.config_entries.async_forward_entry_setup(
+        config_entry, Platform.MEDIA_PLAYER
+    )
+
+    return True
```

Comment:
Can we try to make a connection here? If we can't connect we want to raise ConfigEntryNotReady to show the user in a user-friendly way that we can't work

Reply:
We can store objects in entry.runtime_data and access them again in the media player platform